src/app/components/inventory-page/inventory-page.component.scss

```diff
@@ -0,0 +1,31 @@
+.Container {
+  height: 85vh;
+}
+
+.row {
+  margin: 1%;
+  font-size: 1.7vw;
+  height: 98%;
+}
+
+.section, .sectionEquip, .sectionInventory {
+  max-height: 95%;
+  padding: 1%;
+}
+
+img {
+  height: 90%;
+  width: 100%;
+  background: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.characterName {
+  text-align: center;
+}
+
+h4 {
+  text-transform: uppercase;
+  margin: 2% 0;
+}
```

Comment:
I don't know why but there is a scroll bar even though the number of items do not make it necessary. If it is possible make some changes :D

Reply:
thank you! solved with "overflow-y: auto;" in inventory.scss